examples/lock-app/silabs/SiWx917/src/LockManager.cpp

```diff
@@ -682,15 +682,15 @@ bool LockManager::setLockState(chip::EndpointId endpointId, DlLockState lockStat
     }
 
     // Check the PIN code
-    for (uint8_t i = 0; i < kMaxCredentials; i++)
+    for (uint8_t i = 0; i < kMaxCredentialsPerUser; i++)
```

Comment:
This probably needs to be fixed if we fix the credentials array size... Ranged for loop would be better for that reason.